common/extensions.go

```diff
@@ -16,7 +16,7 @@ type URLStringExtension string
 
 func (s URLStringExtension) RedactSecretQueryParamForLogging() string {
 	u, err := url.Parse(string(s))
-	if err != nil {
+	if err != nil || !strings.HasPrefix(u.Scheme, "http") {
```

Comment:
Just out of curiosity, was there a case where we wanted to avoid the redaction, or is this just to save time?

Reply:
Yeah, this is a fix. If you push a non-URL through logging, it will be encoded as though it's a URL path segment. I intended to avoid that here.